app-i18n/mozc/mozc-2.23.2785.102.1.recipe

```diff
@@ -2,17 +2,17 @@ SUMMARY="Japanese input method editor"
 DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
 This is open source version."
 HOMEPAGE="https://github.com/google/mozc/"
-COPYRIGHT="2010-2016, Google Inc."
+COPYRIGHT="2010-2018, Google Inc."
 LICENSE="BSD (3-clause)"
-REVISION="2"
-fileVersion="v2.20.2677.102.6"
-SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
-CHECKSUM_SHA256="cdb43a88d82db185299a8e396fccfe76ddbbe5a3c6b219220a36b55115cff4ac"
+REVISION="1"
+SOURCE_URI="https://github.com/hanya/mozc/releases/download/v$portVersion/mozc-v$portVersion.tar.bz2"
+CHECKSUM_SHA256="17ce4ee5d6a55678d804890d7e2fa7ffd7f852a9673e3b79671613c9167469f5"
 SOURCE_DIR="mozc"
-if [ $effectiveTargetArchitecture == "x86" ]; then
+if [ "$effectiveTargetArchitecture" = x86 ]; then
 	PATCHES="mozc-$portVersion-x86.patch"
 fi
-SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/0.6.0-20080911/zinnia-tomoe-0.6.0-20080911.tar.bz2"
+zinniaTomoeVersion="0.6.0-20080911"
+SOURCE_URI_2="https://downloads.sourceforge.net/zinnia/zinnia-tomoe-$zinniaTomoeVersion.tar.bz2"
 CHECKSUM_SHA256_2="fd1a264b8aeea0ebec59e8c8765864a9543a75fb2dba349745bb3a3587790927"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Comment:
Oops, I forgot to add an important line!
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

With all the changes, you should be able to call `haikuporter mozc_x86` on x86_gcc2h.
This is what I did, but the compilation fails at some point.

Reply:
I tried to build mozc_x86 on hybrid environment. I met linking error to /boot/system/servers/input_server which required to make input server that inherits BInputServerMethod class.
It looks input_server is compiled as x86_gcc2, I'm not sure it's real. 
So I leave the secondary achitecture still broken.